action.yml

```diff
@@ -15,10 +15,10 @@ inputs:
 runs:
   using: composite
   steps: 
-    - run: "[ -f kubectl-crossplane ] || curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | sh"
+    - run: "[ -f crossplane ] || curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | sh"
```

Comment:
Would using a git reference other than `master` help here? e.g. Pin the curl to `release-1.13` or similar?

Reply:
Yes, that would mean using the old version of the script, which downloaded the binary with the old name, but people setting it the channel to current instead of the default stable would still get the version built from master. Overall, the whole approach feels a bit brittle.